pass headers

bborn2 opened the pull request.

Commit:
add headers

Commit:
add headers

Commit:
add headers

Commit:
update package

Commit:
support post

chrishoage commented:
I'm not sure how I am able to merge this when you've changed the package.json? ~~You've also introduced a breaking change by requiring a root to be set.~~ Never mind you're setting a default. Point about the package.json still stands. though.

Commit:
Fixed response status code

bborn2 committed:
Merge pull request #1 from sh33dafi/response_code

Fixed response status code

utsavdevsharma committed:
New release

New npm release to include last commit https://github.com/bborn2/gulp-connect-proxy-with-headers/commit/2586b1215f801a8db1744f2384636fa8879d3bbe

bborn2 committed:
Merge pull request #2 from PrimeDigitalTech/master

New npm release
